server/server/DBRepositories/MSSQL/MSSQLAdminRepository.cs

```diff
@@ -0,0 +1,67 @@
+﻿using server.Interfaces;
+using server.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace server.DBRepositories
+{
+    public class MSSQLAdminRepository : IRepository<Admin, string>
```

Comment:
As recommendation, it is possible to implement generic repository. That can get access to DbSet as context.Set<EntityType>